llvm/tools/llvm-profdata/llvm-profdata.cpp

```diff
@@ -207,6 +207,12 @@ cl::opt<bool> GenPartialProfile(
     "gen-partial-profile", cl::init(false), cl::Hidden,
     cl::sub(MergeSubcommand),
     cl::desc("Generate a partial profile (only meaningful for -extbinary)"));
+cl::opt<bool> SplitLayout(
+    "split_layout", cl::init(false), cl::Hidden,
```

Comment:
What's the reason to use `split_layout` as opposed to `split-layout`? You can see all other flags are using `-` in their names.

Reply:
done